.gitignore

```diff
@@ -68,5 +68,5 @@ Session.vim
 /cmd/*/.ethereum/
 
 # created for running container
-_assets/compose/mailserver/config.json
+config.json
```

Comment:
```suggestion
mailserver-config.json
```

Reply:
I think I'd just rather move the generated config to the `DATA_PATH` in `/var/tmp/status-go-mail`.